perma_web/perma/tasks.py

```diff
@@ -371,6 +371,21 @@ class HaltCaptureException(Exception):
     thread_list = []
     successful_favicon_urls = []
 
+    def sleep_unless_seconds_passed(seconds):
+        delta = time.time() - start_time
+        if delta < seconds:
+            wait = seconds - delta
+            print("Sleeping for {}s".format(wait))
+            time.sleep(wait)
+
+    # By domain, code to run after the target_url's page onload event.
+    # (Wrap in a lambda function to delay execution.)
+    special_domains = {
+        # Wait for splash page to auto redirect
+        "www.forbes.com": lambda: sleep_unless_seconds_passed(24)
```

Comment:
We have just a few archives that are of "http://forbes.com", wondering if we should drop the "www"?